server/codelogs/codelog_handler.coffee

```diff
@@ -0,0 +1,24 @@
+CodeLog = require './CodeLog'
+Handler = require '../commons/Handler'
+mongoose = require 'mongoose'
+
+CodeLogHandler = class CodeLogHandler extends Handler
+  modelClass: CodeLog
+  makeNewInstance: (req) ->
+    codelog = super(req)
+    codelog.id = req.id
+    codelog.set 'sessionID', req.body.sessionID
+    codelog.set 'level', req.body.level
+    codelog.set 'levelSlug', req.body.levelSlug
+    codelog.set 'userID', req.body.userID
+    codelog.set 'userName', req.body.userName
+    codelog.set 'log', req.body.log
+    codelog.set 'date', new Date().toISOString()
+    codelog.set 'id', req.body.id
```

Comment:
What's this bit about setting `id` here and on line 9? I think that Mongo will set the `_id`, right, and it's `_id`, not `id`.

Also I think we need to take out `date`, that's now automatically set as `created` by the Mongoose schema.